qa/tasks/cephfs/test_exports.py

```diff
@@ -25,6 +25,7 @@ def _wait_subtrees(self, status, rank, test):
     def test_export_pin(self):
         self.fs.set_allow_multimds(True)
         self.fs.set_max_mds(2)
+        self.fs.wait_for_daemons()
```

Comment:
It also checks there are `max_mds` actives?

https://github.com/batrick/ceph/blob/d4870a093ce95cfa5246e470cfa839977aabdbf1/qa/tasks/cephfs/filesystem.py#L612